app/src/main/java/org/torproject/android/ui/onboarding/OnboardingActivity.java

```diff
@@ -2,62 +2,83 @@
 
 import android.content.Context;
 import android.content.Intent;
+import android.content.SharedPreferences;
 import android.os.Bundle;
+
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
+
+import android.util.Log;
 import android.view.View;
+
 import com.github.paolorotolo.appintro.AppIntro;
+
 import org.torproject.android.R;
+import org.torproject.android.service.util.Prefs;
 import org.torproject.android.settings.LocaleHelper;
 import org.torproject.android.ui.AppManagerActivity;
 import org.torproject.android.ui.VPNEnableActivity;
 import org.torproject.android.ui.hiddenservices.permissions.PermissionManager;
 
+import java.util.HashMap;
+import java.util.LinkedList;
```

Comment:
unused

Reply:
Oh, yeah. I forgot to remove it at the end.

Reply:
cool, sorry about how terse my code review comments were earlier ~ fixing UX bugs in anonymity software is really important, this was a good find + fix :+)

Reply:
no issue :)